spec/yaml/swiftnav/sbp/file_io.yaml

```diff
@@ -51,18 +50,39 @@ definitions:
           size: 20
           desc: Name of the file to read from (NULL padded)
 
- - MSG_FILEIO_READ_DIR:
+ - MSG_FILEIO_READ_RESPONSE:
+    id: 0x00A3
+    short_desc: File read from the file system (host <= device)
+    desc: |
+      The file read message reads a certain length (up to 255 bytes)
+      from a given offset into a file, and returns the data in a
+      message where the message length field indicates how many bytes
+      were succesfully read.
+    fields:
+      - offset:
+          type: u32
+          units: bytes
+          desc: File offset
+      - chunk_size:
+          type: u8
+          units: bytes
+          desc: Chunk size read
+      - filename:
```

Comment:
This is missing the actual byte stream from the file.  In keeping with the current design, you would get these fields + the rest of the 256 bytes with a portion of the file.